### ISSUE TITLE
Hide decorative images from assistive tech

### ISSUE THREAD

**Issue by HaSistrunk** [RockefellerArchiveCenter/dimes#173]
## Describe the bug
Several icons are not hidden from screen readers, so the sr reads out the icon name.

## Expected behavior
Decorative images with no necessary information including icons should be hidden from screen readers so that they are ignored.

## Additional context
- RAC logo on homepage: Has `alt=" "`, which should actually be `alt=""` with no space
- Tile type labels in search results: icons indicating collection, person, or organization. SR reads is "image".
- Arrow left icon in "Back to Search" button
- Expand more arrow for collection content collections 
![image](https://user-images.githubusercontent.com/15254041/96311873-70844000-0fd8-11eb-8e69-34e88cbd227d.png)
- Found in icons (sr is reading as "unlabeled image"): 
![image](https://user-images.githubusercontent.com/15254041/96312017-bd681680-0fd8-11eb-9c3f-8d173df6117a.png)
- Sort by search dropdown icon:
![image](https://user-images.githubusercontent.com/15254041/96478442-af4b0d80-1205-11eb-9bcb-0bb9dd6acd9d.png)
